acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/configuration/genesis/QbftValidatorContractConfigFactory.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Hyperledger Besu Contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.tests.acceptance.dsl.node.configuration.genesis;
+
+import static java.util.stream.Collectors.toList;
+
+import org.hyperledger.besu.tests.acceptance.dsl.node.RunnableNode;
+
+import java.math.BigInteger;
+import java.util.Collection;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.tuweni.bytes.Bytes;
+
+public class QbftValidatorContractConfigFactory {
+  private static final String BALANCE = "0";
+  // This is compiled from consensus/qbft/src/integration-test/resources/validator_contract.sol
```

Comment:
If we can't compile dynamically in the test, is it better to comment with a github permalink to the exact version of the contract that won't change over time?